extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -80,9 +84,13 @@ require([
 				comscore: !win.wgDevelEnvironment
 			},
 			autoplay: willAutoplay,
+			captions: {
+				enabled: featuredVideoCookieService.areCaptionsEnabled()
+			},
 			settings: {
 				showAutoplayToggle: true,
-				showQuality: true
+				showQuality: true,
+				showCaptionsToggle: true
```

Comment:
it won't be a toogle soon, should we rename it already?

Reply:
I would rename it when we'll be doing the other solution